pykechain/models/widgets/helpers.py

```diff
@@ -1,6 +1,7 @@
 from typing import Dict, Optional, Union, Text, Tuple, List, Callable
 
-from pykechain.enums import Category, PropertyType, WidgetTitleValue
+from pykechain.enums import Category, PropertyType, WidgetTitleValue, ActivityType, \
+    CardWidgetLinkValue, KEChainPages, CardWidgetKEChainPageLink, LinkTargets, ImageFitValue, CardWidgetImageValue
```

Comment:
Use tuple to import, more explicit, less syntactic
```suggestion
from pykechain.enums import (Category, PropertyType, WidgetTitleValue, ActivityType, 
    CardWidgetLinkValue, KEChainPages, CardWidgetKEChainPageLink, LinkTargets, ImageFitValue, CardWidgetImageValue)
```

Reply:
Fixed